Scholars Portal Journals.js

```diff
@@ -0,0 +1,172 @@
+{
+	"translatorID": "5ac0fd37-5578-4f82-8340-0e135b6336ee",
+	"label": "Scholars Portal Journals",
+	"creator": "Bartek Kawula",
+	"target": "https?://journals[1-2]\\.scholarsportal\\.info/",
```

Comment:
`"^https?://journals\d\\.scholarsportal\\.info/"` (it's supporting 1-4 atm, but there's no harm in making this more future-proof)